cfme/tests/automate/test_git_import.py

```diff
@@ -378,3 +378,68 @@ def test_automate_git_import_case_insensitive(request, appliance):
     domain = appliance.collections.domains.instantiate(name="testdomain")
     request.addfinalizer(domain.delete_if_exists)
     assert domain.exists
+
+
+@pytest.mark.tier(3)
+@pytest.mark.meta(automates=[1391208])
+@pytest.mark.parametrize(
+    ("url", "param_type", "param_value", "connection"),
+    [
+        (
+            GIT_REPO_URL,
```

Comment:
If the value of `url`  for both the parameters is same i.e. `GIT_REPO_URL`, why are you parametrizing it? Why not just directly use the value?

Reply:
Removed parameterization